### PR TITLE
Update pension introduction page

### PR DESCRIPTION
**Pull request by cehsu**
localhost-3001-pension-application-527ez-introduction 1

localhost-3001-pension-application-527ez-introduction 2

### CODE REVIEW

File: src/js/pensions/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -20,18 +20,23 @@ class IntroductionPage extends React.Component {
...
+        <h4>Follow the steps below to apply for a Veterans pension.</h4>
         <div className="process schemaform-process schemaform-process-sip">
           <ol>
             <li className="process-step list-one">
               <div><h5>Prepare</h5></div>
               <div><h6>To fill out this application, you’ll need your:</h6></div>
               <ul>
                 <li>Social Security number or VA file number (required)</li>
-                <li>Military history (required)</li>
+                <li>Military history —— information like service dates, discharge character and rank that's commonly found on the DD214 or other separation documents (required)</li>
```
Comment (jbalboni) [inline src/js/pensions/components/IntroductionPage.jsx]: I don't see this change mentioned in the issue. If it's just in the screenshot, I would want to confirm that it's an intentional change.

File: src/js/pensions/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -20,18 +20,23 @@ class IntroductionPage extends React.Component {
         <FormTitle title="Apply for Veterans pension benefits"/>
         <p>Equal to VA Form 21P-527EZ (Application for Pension).</p>
         <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
           pageList={this.props.route.pageList}
-          resumeOnly
+          startClass="no-text-transform"
```
Comment (jbalboni) [inline src/js/pensions/components/IntroductionPage.jsx]: Does this need to be a prop? Could we just set `no-text-transform` all the time?

File: src/js/pensions/components/IntroductionPage.jsx [hunk 3]
```diff
@@ -49,21 +54,16 @@ class IntroductionPage extends React.Component {
...
+              <a href="/pension/aid-attendance-housebound/" target="_blank">Learn more about Aid and Attendance and Housebound Benefits</a>.<br/><br/>
               <p><strong>What if I need help filling out my application?</strong> An accredited representative with a Veterans Service Organization (VSO) can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Find an accredited representative</a>.</p>
               <h6>Learn about Veterans pension rates</h6>
               <a href="/pension/rates" target="_blank">Find out how we decide pension rates.</a>
             </li>
             <li className="process-step list-two">
               <div><h5>Apply</h5></div>
               <p>Complete this pension benefits form.</p>
-              <p>After submitting the form, you'll get a confirmation message. You can print this for your records.</p>
+              <p>After submitting the form, you'll get a confirmation message that you can print.</p>
```
Comment (jbalboni) [inline src/js/pensions/components/IntroductionPage.jsx]: Same question as above, I don't think this is meant to be changed.

**Review by jbalboni — COMMENTED** [review]
Looks good, but I have a couple questions.

File: src/js/pensions/components/IntroductionPage.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/pensions/components/IntroductionPage.jsx]: Thanks @jbalboni, good point, could @peggygannon @stacyhlinn please let us know?

File: src/js/pensions/components/IntroductionPage.jsx (hunk 3 above)
Comment (cehsu) [reply src/js/pensions/components/IntroductionPage.jsx]: Thanks @jbalboni, good point, could @peggygannon @stacyhlinn please let us know?

**Review by stacyhlinn — APPROVED** [review]
LGTM!

File: src/js/pensions/components/IntroductionPage.jsx (hunk 3 above)
Comment (stacyhlinn) [reply src/js/pensions/components/IntroductionPage.jsx]: @cehsu whoops just approved this before seeing the question. Yeah this was not meant to be changed. We should keep the copy as is on the current page. I might have been using a dated screenshot when creating this mockup.

File: src/js/pensions/components/IntroductionPage.jsx (hunk 1 above)
Comment (peggygannon) [reply src/js/pensions/components/IntroductionPage.jsx]: @cehsu Thanks for checking. The line should be:

Military history (required) ....as it is on the current page.

File: src/js/pensions/components/IntroductionPage.jsx (hunk 3 above)
Comment (peggygannon) [reply src/js/pensions/components/IntroductionPage.jsx]: @cehsu Hello, the line "You can print print this for your records." Should be there as we have on the current page.

File: src/js/pensions/components/IntroductionPage.jsx (hunk 2 above)
Comment (cehsu) [reply src/js/pensions/components/IntroductionPage.jsx]: Great idea, the class necessitating this has been removed, and instances of non-title-cased text, title cased.

File: src/js/pensions/components/IntroductionPage.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/pensions/components/IntroductionPage.jsx]: Thanks @peggygannon, this text has been reverted to current version.

File: src/js/pensions/components/IntroductionPage.jsx (hunk 3 above)
Comment (cehsu) [reply src/js/pensions/components/IntroductionPage.jsx]: Thanks @peggygannon, this text has been reverted to current version.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**